libbeat/tests/system/config/libbeat.yml.j2

```diff
@@ -0,0 +1,98 @@
+#================================ General =====================================
+
+# The name of the shipper that publishes the network data. It can be used to group
+# all the transactions sent by a single shipper in the web interface.
+# If this options is not defined, the hostname is used.
+name: {{shipper_name}}
+
+# The tags of the shipper are included in their own field with each
+# transaction published. Tags make it easy to group servers by different
+# logical properties.
+tags: [
+  {%- if agent_tags -%}
+      {%- for tag in agent_tags -%}
+          "{{ tag }}"
+          {%- if not loop.last %}, {% endif -%}
+      {%- endfor -%}
+  {%- endif -%}
+]
+
+{% if geoip_paths is not none %}
+geoip:
```

Comment:
What's this for? AFAIK we don't have any builtin geoip enrichment.

Reply:
That was copy paste from filebeat. Removed now as we don't use it anymore.